### ISSUE TITLE
Invincible To Used Items/Phases

### ISSUE THREAD

**Issue by Latenciiii** [kangarko/CoreArena#486]
Hello,

-Would it be possible to make the items such as the rocket or hellfire so they dont harm players? Ive got quite small arenas and its hard to avoid the explosions!
-The phases system, this is probably implemented im just having issues figuring it out, i would like it so for example;

Phase 1 - 50 mobs
Phase 2 - 75 mobs
Phase 3 - 100 mobs - mini boss (but phase ends up on boss kill but he doesnt spawn until all 100 mobs have been killed)
Phase 4 - 150 mobs
Phase 5 - 200 mobs + main boss
_Then arena ends._

That may be possible and if you could point me in to a detailed how to guide i just cant get my head around it xD

Many Thanks!

**Comment by Latenciiii** [comment]
Hi,

i guess its because im using it as a mob arena rather than a player arena?
Just the option to toggle that kind of damage would be cool!

Thank you for that!

**Comment by Latenciiii** [comment]
Hi,

Sorry to email you directly but I couldnt find anywhere else to contact.

Am I able to get the download links for the 2 maps that come along with
CoreArena? I have had them in the past but stopped playing!

Thank you
Kyle

**Comment by kangarko** [comment]
Yes sure, you will see them if you join our discord server at mineacademy.org/discord or send a private email to support@mineacademy.org
